src/main/webapp/pages/projects/project_samples.html

```diff
@@ -171,14 +171,15 @@
                                 </a>
                             </li>
                             <li th:if="${linkerAvailable}" role="menuitem">
-                                <a href="#" class="js-sample-tool-btn t-linker-btn"
-                                   data-toggle="tooltip" data-enabled-at="0"
-                                   data:url="@{/projects/{projectId}/templates/linker-modal(projectId=${project.getId()})}"
-                                   data:script="@{/dist/js/project-samples-linker.bundle.js}"
-                                   data:associated-msg="#{project.samples.only-for-project-tooltip}">
-                                    <i class="fa fa-terminal fa-fw spaced-right__sm" aria-hidden="true"></i>
-                                    <th:block th:text="#{project.samples.export.linker}"/>
-                                </a>
+                                <a href="#"><span id="linker"></span></a>
+<!--                                <a href="#" class="js-sample-tool-btn t-linker-btn"-->
```

Comment:
Same as above. Can be removed

Reply:
Removed in 24c9db3c703d87b5a0e071c19cb15a6e79048905